txscript/standard_test.go

```diff
@@ -1013,3 +1013,28 @@ func TestStringifyClass(t *testing.T) {
 		}
 	}
 }
+
+// TestNullDataScript tests whether NullDataScript returns a valid script.
+func TestNullDataScript(t *testing.T) {
+	data := []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
```

Comment:
Thanks for all your advice but I'm still pretty confused. Your example "01" => "6a51" is what the code you suggested produces, but the result is not recognized as a NullDataTy when I use GetScriptClass. It returns NonStandardTy. Is that correct? If so, then NullDataScript should probably return an error rather than 6a51.